src/api.rs

```diff
@@ -1685,6 +1666,72 @@ impl<'a> AuthenticatedApi<'a> {
             resp.convert()
         }
     }
+
+    pub fn region_specific<'b>(&'a self, org: &'b str) -> RegionSpecificApi<'a, 'b> {
+        let base_url = self.api.config.get_base_url();
+        if base_url.is_err()
+            || base_url.expect("base_url should not be error") != DEFAULT_URL.trim_end_matches('/')
```

Comment:
Is this any chance this error is surfaced to the user? If so, should we rephrase the expect message to be something more along the lines of "Invalid Base URL provided"?

Reply:
This error should never occur because the `base_url.is_err()` check before will short circuit. `expect` is equivalent to `unwrap` just allows an error message, but I prefer to use `expect` mostly just to document why it is okay to unwrap the value.